commercetools-models/src/main/java/io/sphere/sdk/apiclient/expansion/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * {@doc.gen summary subscriptions}
+ */
+package io.sphere.sdk.apiclient.expansion;
```

Comment:
Not needed!?

Reply:
can be removed, was following the generated code pattern there.